Makefile

```diff
@@ -456,8 +457,19 @@ __test-image:
 		--build-arg YQ_VERSION=$(YQ_VERSION) \
 		--build-arg BATS_VERSION=$(BATS_VERSION) \
 		--build-arg ORAS_VERSION=$(ORAS_VERSION) \
-		--build-arg KUSTOMIZE_VERSION=$(KUSTOMIZE_VERSION) \
-		--build-arg KUBEBUILDER_VERSION=$(KUBEBUILDER_VERSION)
+		--build-arg KUSTOMIZE_VERSION=$(KUSTOMIZE_VERSION)
+
+## Location to install dependencies to
```

Comment:
fyi, I can see `make test` work locally on this fork now 🥇 ; thanks for fixing this !!